src/Features/Core/Portable/NavigateTo/NavigateToSearcher.cs

```diff
@@ -161,35 +161,29 @@ private async Task SearchAllProjectsAsync()
             if (_searchCurrentDocument && _currentDocument?.Project != project)
                 return;
 
-            var cacheService = project.Solution.Services.CacheService;
-            if (cacheService != null)
```

Comment:
❔ Is there ever a case where this was null and we were using that fact to avoid some amount of unnecessary work?